### DISCUSSION TITLE
Category is leading to the homepage not the parent listing page.

### DISCUSSION THREAD

**Discussion by rohitfarmer** [quarto-dev/quarto-cli#9685] [Q&A]
### Description

Hi there, I have three listing pages on my website, https://outdoorsyindians.com/, under the Posts tab. The same three listings are also posted in different sections with ID calls on the home page. So when I open a blog post like https://outdoorsyindians.com/camping/usa/maryland/elk-neck-state-park/ and click on the category, it leads me to the homepage of the website, not to its parent listing page, https://outdoorsyindians.com/camping. How can I fix it? Any suggestions would be helpful. Thanks!

**Comment by mcanouil** [comment]
Would it be possible for you to make a small reproducible example and to share it as a Git repository?
